### PR TITLE
Adjust README for packaging work

### PR DESCRIPTION
**Pull request by bbowman**

### CODE REVIEW

**Comment by bbowman** [comment]
https://github.com/bbowman/WinObjC/tree/bbowman/packaging_documentation if you want to see it as markdown. A lot of the badges obviously aren't lit up yet. I don't know why the Github release one isn't.

File: README.md [hunk 1]
```diff
@@ -1,14 +1,31 @@
...
-- The latest pre-built Windows Bridge for iOS SDK, that you can download [here](https://aka.ms/winobjc_latest). Click on the **winobjc.zip** file under *Downloads* to start the download and extract it to the local directory of your choice (example: C:\winobjc) when complete.
 - **Windows 10**, build 10586 or higher. Validate your version number [here](http://windows.microsoft.com/en-US/windows/which-operating-system).
-- **Visual Studio 2015 Update 3** with Windows developer tools. Visual Studio 2015 Community is available for free [here](https://dev.windows.com/downloads). Select (at least) the following components during installation:
+- **Visual Studio 2017** with Windows developer tools. Visual Studio 2017 Community is available for free [here](https://dev.windows.com/downloads). Select (at least) the following components during installation:
    1. Programming Languages -> Visual C++
    2. Universal Windows App Development Tools (all)
-   
+
+To get started with an existing Xcode Project, you will also need:
+- **Chocolately**, a package manager for windows. See how to install the latest version [here] (https://chocolatey.org/)
```
Comment (DHowett-MSFT) [inline README.md]: Choco-lately!

File: README.md [hunk 2]
```diff
@@ -17,15 +34,15 @@ If you're having trouble downloading & installing the bridge, see the [Known Iss
 ### Importing your Xcode Project
 When using the bridge, the first thing you'll want to do is generate a Visual Studio solution from your Xcode project:
 
-1. Open the Windows Command Prompt (type **cmd** in the Start Menu to find it) and navigate to your Xcode project directory using the **cd** command. Note: use the directory containing your .workspace or .xcodeproj folder.
+1. Open the Windows PowerShell (type **powershell** in the Start Menu to find it) and navigate to your Xcode project directory using the **cd** command. Note: use the directory containing your .workspace or .xcodeproj folder.
```
Comment (DHowett-MSFT) [inline README.md]: nit: remove `the` from `the Windows PowerShell`

File: README.md [hunk 3]
```diff
@@ -1,14 +1,31 @@
...
+|**WinObjC.Frameworks**|[![Frameworks][nuget-frmwrk-badge]][nuget-frmwrk] |[![Frameworks][nuget-frmwrk-badge-pre]][nuget-frmwrk]|
+
 ## Download and Installation
 To use the bridge you'll need:
-- The latest pre-built Windows Bridge for iOS SDK, that you can download [here](https://aka.ms/winobjc_latest). Click on the **winobjc.zip** file under *Downloads* to start the download and extract it to the local directory of your choice (example: C:\winobjc) when complete.
 - **Windows 10**, build 10586 or higher. Validate your version number [here](http://windows.microsoft.com/en-US/windows/which-operating-system).
-- **Visual Studio 2015 Update 3** with Windows developer tools. Visual Studio 2015 Community is available for free [here](https://dev.windows.com/downloads). Select (at least) the following components during installation:
+- **Visual Studio 2017** with Windows developer tools. Visual Studio 2017 Community is available for free [here](https://dev.windows.com/downloads). Select (at least) the following components during installation:
    1. Programming Languages -> Visual C++
    2. Universal Windows App Development Tools (all)
```
Comment (memontic-ms) [inline README.md]: Need to be updated to reflect 2017 and the new installer. Should be tested out with a clean install. This should look more like:
Code tools -> NuGet package manager
Compilers, build tools, and runtimes -> ? Most of those?
SDKs, libraries, and frameworks -> Windows 10 SDK (10.0.10586.0 and 10.0.14393.0 if only building build.sln. 10.0.10240 can be used for some projects under tools)

Comment (memontic-ms) [inline README.md]: Okay, this is the Download and Installation section. Seems like this should be more of a "Add WinObjC.Framworks package reference to your project" section.
The full list of VS install options for WinObjC development should be included under the fancy Advanced Installation for Contributors and Ninjas section.

---
In reply to: [105712228](https://github.com/Microsoft/WinObjC/pull/2211#discussion_r105712228) [](ancestors = 105712228)

File: README.md [hunk 4]
```diff
@@ -1,14 +1,31 @@
...
-   
+
+To get started with an existing Xcode Project, you will also need:
+- **Chocolatey**, a package manager for windows. See how to install the latest version [here] (https://chocolatey.org/)
+- **WinObjC.Tools**, the command line tools for WinObjC. Run the below command from **powershell** to get the latest version:
```
Comment (memontic-ms) [inline README.md]: >**powershell** [](start = 84, length = 14)

Needs to be Administrator Powershell (at least for me it requires Admin)

**Pull request merged**